pkg/api/api.go

```diff
@@ -249,6 +249,11 @@ func (hs *HTTPServer) registerRoutes() {
 		apiRoute.Get("/plugins/:pluginId/settings", Wrap(GetPluginSettingByID))
 		apiRoute.Get("/plugins/:pluginId/markdown/:name", Wrap(GetPluginMarkdown))
 
+		// Reload Grafana components
+		apiRoute.Group("/reload", func(reloadRoute routing.RouteRegister) {
+			reloadRoute.Post("/ldapcfg", Wrap(ReloadLdapCfg))
```

Comment:
I am thinking about a bit different url structure here. We already have a `/api/admin` path and also `/api/admin/settings`. Seems to this would make sense as `POST /api/admin/settings/ldap/reload`. That way we can keep expanding it keeping structure `/api/admin/settings/<section>/reload` while also having other verbs available for `/api/admin/settings/<section>` paths. @bergquist what do you think?

Reply:
Sounds good to me

Reply:
Thanks guys. Will do it tonight or tomorrow.